src/index.ts

```diff
@@ -395,7 +485,7 @@ export default class Header {
     ];
 
     return this._settings.levels ? availableLevels.filter(
-      l => this._settings.levels.includes(l.number)
+      l => this._settings.levels?.includes(l.number)
```

Comment:
? seems unnecessary, since you're checking for emptiness before it

Reply:
replace it with '!' to insure the non-null assertion, as it will not be build